manage.py

```diff
@@ -1482,39 +1482,71 @@ def _validate_memory(val):
 
 
 def _add_rule_table_subparser(subparsers):
-    """Add the rule database helper subparser: manage.py rule-table [subcommand] [options]"""
-    usage = 'manage.py rule-table [subcommand] [options]'
+    """Add the rule database helper subparser: manage.py rule-staging [subcommand] [options]"""
+    usage = 'manage.py rule-staging [subcommand] [options]'
     description = """
 StreamAlertCLI v{}
 Print the status of or update remote StreamAlert rule information within the rule database
 
 Available Subcommands:
 
-    manage.py rule-table status          List the current staging status from the rules databse
-    manage.py rule-table stage           Stage the rules provided in a space-separated list
-    manage.py rule-table unstage         Unstage the rules provided in a space-separated list
+    manage.py rule-staging enable          Enable or disable the rule staging feature
+    manage.py rule-staging status          List the current staging status from the rules databse
+    manage.py rule-staging stage           Stage the rules provided in a space-separated list
+    manage.py rule-staging unstage         Unstage the rules provided in a space-separated list
 
 """.format(version)
 
-    rule_table_parser = _generate_subparser(subparsers, 'rule-table', usage, description)
+    rule_table_parser = _generate_subparser(subparsers, 'rule-staging', usage, description)
 
     rule_table_subparsers = rule_table_parser.add_subparsers()
 
+    _add_rule_table_enable_subparser(rule_table_subparsers)
     _add_rule_table_status_subparser(rule_table_subparsers)
     _add_rule_table_stage_subparser(rule_table_subparsers)
     _add_rule_table_unstage_subparser(rule_table_subparsers)
 
+def _add_rule_table_enable_subparser(subparsers):
```

Comment:
Do you think it is more reasonable to rename this method and following three methods wiht prefix `_add_rule_table` to `_add_rule_staging` now as well. Because we rename `rule-table` to `rule-staging` in cli.

Reply:
good catch!! sorry I thought I had addressed all of these!